### ISSUE TITLE
Google SignIn Error : Extra argument 'sourceApplication' in call at 4th configuration step

### ISSUE THREAD

**Issue by elnoidelkargol** [firebase/firebase-ios-sdk#3906]
### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 10.3
  * Firebase SDK version: 6.8.1
  * Firebase Component: Auth ( Google SignIn)
  * Component version: GoogleSignIn (5.0.0)

### [REQUIRED] Step 3: Describe the problem
When I try to configurate GoogleSignIn I obtain the error "Extra Argument ' source Application' in call. This error appears in the **4th step** on you Firebase Guide Configuration ( [https://firebase.google.com/docs/auth/ios/google-signin?authuser=0](https://firebase.google.com/docs/auth/ios/google-signin?authuser=0))  

when I add the following code 

`
@available(iOS 9.0, *)
func application(_ application: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any])
  -> Bool {
    return GIDSignIn.sharedInstance().handle(url,
                            sourceApplication:options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
                            annotation: [:])
}
`

#### Steps to reproduce:
 
Reproduce the 4th step of Firebase Auth with Google SignIn guide
 [https://firebase.google.com/docs/auth/ios/google-signin?authuser=0](https://firebase.google.com/docs/auth/ios/google-signin?authuser=0)

#### Relevant Code:

```
@available(iOS 9.0, *)
func application(_ application: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any])
  -> Bool {
    return GIDSignIn.sharedInstance().handle(url,
                            sourceApplication:options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
                            annotation: [:])
}
```

**Comment by morganchen12** [comment]
This is a bug in the docs. With GIDSignIn 5.0 and above, you don't need the `sourceApplication` parameter. Use [this method](https://developers.google.com/identity/sign-in/ios/reference/Classes/GIDSignIn#/c:objc(cs)GIDSignIn(im)handleURL:) instead.

**Comment by parschina** [comment]
[واردات از چین](https://parschina.ir/)

**Comment by morganchen12** [comment]
Fixed in cr/284802577 (internal link). The changes will be available publicly sometime soon.
